.github/workflows/tests.yml

```diff
@@ -80,14 +156,26 @@ jobs:
         with:
           fetch-depth: 0
       - id: changes
+        name: "Compute changed files"
+        shell: bash
+        run: |
+          if [[ "${{ github.event_name }}" == "pull_request" ]] && [[ "${{ github.actor }}" != "dependabot[bot]" ]]; then
+              changes () { git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }} ; }
+              echo "changed files? $(changes)"
+              echo "::set-output name=js::$(changes | grep -E 'apps/site/assets/js/.*\.jsx?' | xargs)"
+              echo "::set-output name=ts::$(changes | grep -E 'apps/site/assets/ts/.*\.tsx?' | xargs)"
+              echo "::set-output name=scss::$(changes | grep -E 'apps/site/assets/css/.*\.scss' | xargs)"
+              echo "::set-output name=ex::$(changes | grep -E '.*(\.exs?|\.eex)' | xargs)"
+          else
+              echo "Not a PR by a dev. Use placeholder value so all tests run"
+              echo "::set-output name=js::skip"
```

Comment:
Can you change the value to "all" instead of "skip"? Since I believe the intent is to run all the tests, not skip all the tests.